File: RC1.5/examples/example1.cpp
#include <iostream>
#include "DayTime.hpp"

using namespace std;
using namespace gpstk;

int main(int argc, char* argv[])
{

   try {
      
      DayTime time;

      cout << "Hello world!" << endl;
      cout << "   The current GPS week is " << time.GPSfullweek() << endl;
      cout << "   The day of the GPS week is " << time.GPSday() << endl;
      cout << "   The seconds of the GPS week is " << time.GPSsecond() << endl;

   }
   catch( Exception error)
   {
      cout << error << endl;
      exit(-1);
   }

   exit(0);
}

File: branches/rinex3_dev/apps/bindings/python/examples/example1.py
#!/usr/bin/python

import sys
sys.path.append("..")
from gpstkPython import *

try:
    time = DayTime()

    print 'Hello world!'
    print 'The current GPS week is', time.GPSfullweek()

    print 'The day of the GPS week is', time.GPSday()

    print 'The seconds of the GPS week is', time.GPSsecond()

except Exception:
    print 'Error'


